packages/timeline-state-resolver/src/devices/vizMSE.ts

```diff
@@ -732,27 +760,51 @@ export class VizMSEDevice extends DeviceWithState<VizMSEState, DeviceOptionsVizM
 
 		try {
 			if (this._vizmseManager) {
-				if (cmd.type === VizMSECommandType.PREPARE_ELEMENT) {
-					await this._vizmseManager.prepareElement(cmd)
-				} else if (cmd.type === VizMSECommandType.CUE_ELEMENT) {
-					await this._vizmseManager.cueElement(cmd)
-				} else if (cmd.type === VizMSECommandType.TAKE_ELEMENT) {
-					await this._vizmseManager.takeElement(cmd)
-				} else if (cmd.type === VizMSECommandType.TAKEOUT_ELEMENT) {
-					await this._vizmseManager.takeoutElement(cmd)
-				} else if (cmd.type === VizMSECommandType.CONTINUE_ELEMENT) {
-					await this._vizmseManager.continueElement(cmd)
-				} else if (cmd.type === VizMSECommandType.CONTINUE_ELEMENT_REVERSE) {
-					await this._vizmseManager.continueElementReverse(cmd)
-				} else if (cmd.type === VizMSECommandType.LOAD_ALL_ELEMENTS) {
-					await this._vizmseManager.loadAllElements(cmd)
-				} else if (cmd.type === VizMSECommandType.CLEAR_ALL_ELEMENTS) {
-					await this._vizmseManager.clearAll(cmd)
-				} else if (cmd.type === VizMSECommandType.CLEAR_ALL_ENGINES) {
-					await this._vizmseManager.clearEngines(cmd)
-				} else {
-					// @ts-ignore never
-					throw new Error(`Unsupported command type "${cmd.type}"`)
+				switch (cmd.type) {
```

Comment:
Note: I'm not a fan of either construction. In the future this could use some OOP.